README.md

```diff
@@ -889,6 +891,71 @@ If these failures occur, add the warning to the list of permitted warnings found
 in [`AzulTestCase`](test/azul_test_case.py) and commit the modifications. 
 
 
+## macOS Big Sur: pyenv build failed
+
+[pyenv Big Sur GitHub issue](https://github.com/pyenv/pyenv/issues/1740)
+
+macOS Big Sur users may hit a `build failed` error when installing a new pyenv.
+To remedy this there is a patch file which runs the installer compatible with 
+macOS Big Sur (11.x).
+
+First is to install zlib and bzip2 using brew:
+```
+brew install zlib bzip2
+OR
+brew reinstall zlib bzip2
+```
+
+The output should tell you the steps to editing the path variables LDFLAGS and
+CPPFLAGS, however the following command should suffice:
+```
+export LDFLAGS="-L/usr/local/opt/zlib/lib -L/usr/local/opt/bzip2/lib"
+export CPPFLAGS="-I/usr/local/opt/zlib/include -I/usr/local/opt/bzip2/include"
+```
+The above command can be placed in the `~/.bash_profile` to make the change persistent
+
+The command to install python 3.8.3 using the patch file is as follows:
+```
+pyenv install --patch 3.8.3 < <(curl -sSL https://github.com/python/cpython/commit/8ea6353.patch)
+```
+
+If this failure still occurs then run the following lines and retry installing python 3.8.3:
+```
+sudo rm -rf /Library/Developer/CommandLineTools
+xcode-select --install
+```
+
+## macOS Big Sur: cryptography library fails to install
+
+If the cryptography package fails to install through `make requirements`,
+install OpenSSL through brew (`brew install openssl`) and use the following lines to
+edit the path variable:
+```
+export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
+export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"
+```
+The above command can be placed in the `~/.bash_profile` to make the change persistent
+
+The above lines can be used in tandem with the LDFLAGS and CPPFLAGS from
+installing zlib and bzip2. The lines to use then would be as follows:
+```
+export LDFLAGS="-L/usr/local/opt/zlib/lib -L/usr/local/opt/bzip2/lib -L/usr/local/opt/openssl@1.1/lib"
+export CPPFLAGS="-I/usr/local/opt/zlib/include -I/usr/local/opt/bzip2/include -I/usr/local/opt/openssl@1.1/include"
+```
+The above command can be placed in the `~/.bash_profile` to make the change persistent
+
+
+## macOS Big Sur: `pip install grpcio` fails
```

Comment:
This issue was more related to wheels not being able to be located for Big Sur. Because old version of MacOS return version 10.x when asked what version they are using this caused problems because internally Big Sur is 11.x. Setting this envvar forces the return value to be 10.16, w/ 10.15 being the last release that was before Big Sur.


It happened to be that we found the issue with grpcio but that's only because the specific version that we require in the requirements.txt is missing a wheel for 11.x. If we change the requirements in the future there might be a different failure.